src/platform/utilities/feature-toggles/flipper-client.js

```diff
@@ -71,7 +74,49 @@ function FlipperClient({
       }
     }
     */
-    const { data } = await _fetchToggleValues();
+    let data;
+    const queryParams = new URLSearchParams(window.location.search);
+    const isToggleCacheDisabled =
+      queryParams.get('disableFlipperCache') === 'true';
```

Comment:
We'll probably want to disable it for a day or maybe indefinitely with another param to turn it back on. Thoughts?

Reply:
I like it!